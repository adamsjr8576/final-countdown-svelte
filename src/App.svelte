<script>
	import { categoryOptions, selectedOption } from './stores.js';
	import CategorySelector from './CategorySelector.svelte';

</script>

<main>
	<header>
		<h1>Harvard Art Museum</h1>
	</header>
	{#if $selectedOption.length === 0}
	<CategorySelector />
	{/if}
</main>

<style>
	main {
		text-align: center;
		padding: 1em;
		width: 100%;
		display: flex;
		flex-direction: column;
		align-items: center;
		justify-content: space-between;
	}

	header {
		width: 100%;
		height: 100px;
	}

	h1 {
		color: #ff3e00;
		text-transform: uppercase;
		font-size: 4em;
		font-weight: 100;
		margin: 0;
		width: 100%;
		text-decoration: underline;
	}

</style>
