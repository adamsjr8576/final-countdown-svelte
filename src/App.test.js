import { render, fireEvent } from '@testing-library/svelte';
import App from './App.svelte';

describe('App', () => {
  it('testing', () => {
    const wrapper = render(App);

    expect(wrapper).toMatchSnapshot();
  });
});
