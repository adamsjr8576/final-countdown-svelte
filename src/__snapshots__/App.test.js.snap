// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App testing 1`] = `
Object {
  "component": App {
    "$$": Object {
      "after_update": Array [],
      "before_update": Array [],
      "bound": Object {},
      "callbacks": Object {},
      "context": Map {},
      "ctx": Array [
        false,
        "",
        Array [],
      ],
      "dirty": Array [
        -1,
      ],
      "fragment": Object {
        "c": [Function],
        "d": [Function],
        "i": [Function],
        "l": [Function],
        "m": [Function],
        "o": [Function],
        "p": [Function],
      },
      "not_equal": [Function],
      "on_destroy": Array [
        [Function],
        [Function],
        [Function],
        [Function],
      ],
      "on_mount": Array [],
      "props": Object {},
      "update": [Function],
    },
    "$capture_state": [Function],
  },
  "container": <body>
    <div>
      <main
        class="svelte-gzq4hi"
      >
        <header
          class="svelte-gzq4hi"
        >
          <h1
            class="svelte-gzq4hi"
          >
            Harvard Art Museum
          </h1>
        </header>
         
         
        <h4
          class="svelte-17ldt0s"
        >
          Please select a category below:
        </h4>
         
        <section
          class="svelte-17ldt0s"
        >
          <article
            class="period-article svelte-17ldt0s"
            data-testid="period-article"
          >
            <h2
              class="svelte-17ldt0s"
            >
              Period
            </h2>
             
            <p
              class="svelte-17ldt0s"
            >
              Contains the periods used to describe items
    in the Harvard Art Museums collections.
            </p>
          </article>
           
          <article
            class="place-article svelte-17ldt0s"
            data-testid="place-article"
          >
            <h2
              class="svelte-17ldt0s"
            >
              Place
            </h2>
             
            <p
              class="svelte-17ldt0s"
            >
              Contains the location used to describe items
    in the Harvard Art Museums collections.
            </p>
          </article>
           
          <article
            class="classification-article svelte-17ldt0s"
            data-testid="classification-article"
          >
            <h2
              class="svelte-17ldt0s"
            >
              Classification
            </h2>
             
            <p
              class="svelte-17ldt0s"
            >
              Contains the Classifications used to describe
    item in the Harvard Art Museums collections.
            </p>
          </article>
           
          <article
            class="gallery-article svelte-17ldt0s"
            data-testid="gallery-article"
          >
            <h2
              class="svelte-17ldt0s"
            >
              Gallery
            </h2>
             
            <p
              class="svelte-17ldt0s"
            >
              Contains the Galleries and their items
    in the Harvard Art Museums collections.
            </p>
          </article>
        </section>
         
         
      </main>
    </div>
  </body>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
